scripts/install-flux.sh

```diff
@@ -3,10 +3,21 @@ set -ex
 
 ENV=$1
 CLUSTER_NAME=$2
-GIT_REPO=${3:-https://weaveworks.github.io/flux}
-VALUES=${4-deployments/weave-flux/values.yaml}
+VAULT_NAME=$3
+ENABLE_HELM_TLS=$4
+GIT_REPO=${5:-https://weaveworks.github.io/flux}
```

Comment:
i don't think you should be able to override these, it's just going to cause issues when someone adds a new param, i.e. you updating this is going to break it